specification/communication/data-plane/CallAutomation/stable/2023-03-06/communicationservicescallautomation.json

```
@@ -0,0 +1,2285 @@
+          "description": "Defines options for recognition."
+        },
+        "operationContext": {
+          "description": "The value to identify context of the operation.",
+          "type": "string"
+        }
+      }
+    },
+    "RecognizeOptions": {
+      "required": [
+        "targetParticipant"
+      ],
+      "type": "object",
+      "properties": {
+        "interruptPrompt": {
+          "description": "Determines if we interrupt the prompt and start recognizing.",
+          "type": "boolean"
+        },
+        "initialSilenceTimeoutInSeconds": {
+          "format": "int32",
+          "description": "Time to wait for first input after prompt (if any).",
+          "maximum": 300,
+          "minimum": 0,
+          "type": "integer"
+        },
+        "targetParticipant": {
+          "$ref": "#/definitions/CommunicationIdentifierModel",
+          "description": "Target participant of DTMF tone recognition."
+        },
+        "dtmfOptions": {
+          "$ref": "#/definitions/DtmfOptions",
+          "description": "Defines configurations for DTMF."
+        }
+      }
+    },
+    "Tone": {
+      "enum": [
+        "zero",
+        "one",
+        "two",
+        "three",
+        "four",
+        "five",
+        "six",
+        "seven",
+        "eight",
+        "nine",
+        "a",
+        "b",
+        "c",
+        "d",
+        "pound",
+        "asterisk"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "Tone",
+        "modelAsString": true
+      }
+    },
+    "DtmfOptions": {
+      "description": "Options for DTMF recognition",
+      "type": "object",
+      "properties": {
+        "interToneTimeoutInSeconds": {
+          "format": "int32",
+          "description": "Time to wait between DTMF inputs to stop recognizing.",
+          "maximum": 60,
+          "minimum": 1,
+          "type": "integer"
+        },
+        "maxTonesToCollect": {
+          "format": "int32",
+          "description": "Maximum number of DTMF tones to be collected.",
+          "type": "integer"
+        },
+        "stopTones": {
+          "description": "List of tones that will stop recognizing.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Tone"
+          }
+        }
+      }
+    },
+    "GetParticipantsResponse": {
+      "description": "The response payload for getting participants of the call.",
+      "type": "object",
+      "properties": {
+        "values": {
+          "description": "List of the current participants in the call.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CallParticipant"
+          }
+        },
+        "nextLink": {
+          "description": "Continue of the list of participants",
+          "type": "string"
+        }
+      }
+    },
+    "CallParticipant": {
+      "description": "Contract model of an ACS call participant",
```

It doesn't need to be an ACS call, right? Also we're avoiding the ACS acronym anywhere public because it's ambiguous and branding guidelines say to avoid acronyms in general.
```suggestion
      "description": "A call participant.",
```

---

That is true - i will make an update. Also i will go through to make sure i exclude all ACS acronym.